src/components/album/AlbumChat.js

```diff
@@ -1,40 +1,66 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import io from 'socket.io-client';
+import { useAuth } from '@akosasante/react-auth-context';
 
 const AlbumChat = ({ apiUrl }) => {
-  useEffect(() => {
-    const socket = io(process.env.REACT_APP_SOCKET_BASE_PATH);
+  const [message, setMessage] = useState('');
+  const [socket, setSocket] = useState(null);
+  const regexPattern = /\/albums\/([a-zA-Z0-9]+)/;
+  const match = apiUrl.match(regexPattern);
+  const albumId = match ? match[1] : null;
 
-    // Subscribe to the album chat channel using albumId
-    const regexPattern = /\/albums\/([a-zA-Z0-9]+)/;
-    const match = apiUrl.match(regexPattern);
-    if (match) {
-      // Extract albumId from the URL
-      const albumId = match[1];
-      // Сonnection is established with the Socket.io server
-      socket.on('connect', () => {
-        // Request to the server to join the album chat
-        console.log('Connected to WebSocket server');
-        socket.emit('join:album_chat', albumId);
+  const { user: loggedInUser } = useAuth();
+  const userId = loggedInUser ? loggedInUser.user.id : null;
```

Comment:
This works; we could also write 

```suggestion
  const userId = loggedInUser?.user?.id;
```